docs/development/vscode_debugger.rst

```diff
@@ -50,7 +50,7 @@ Run the tests with the following. (Again, this example is strictly about the E2E
 
 .. code-block:: bash
 
-    TAG=latest DEBUG_SERVICES=grapl_e2e_tests dobi run-e2e-tests
+    DEBUG_SERVICES=grapl_e2e_tests make test-e2e
```

Comment:
BTW if you'd like me to test this still works lmk

Reply:
Yes, please. Thanks!

Reply:
Glad I did, it was slightly broken :)